apps/portal/components/containers/dashboardLayout/elements/SidebarMenuItems/Favorite.tsx

```
@@ -0,0 +1,114 @@
+        <div onClick={handleToggle} className="tw-cursor-pointer tw-mr-2">
+          <AppSvg path="/sidebar/new/starred.svg" size="20px" />
+        </div>
+        <Link href={href} className={styles.title}>
+          {title}
+        </Link>
+        <div
+          onClick={(e) => {
+            e.preventDefault();
+            handleToggle(e);
+          }}
+        >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)